### PR TITLE
update support matrix

### PR DESCRIPTION
**Pull request by raunakkumar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,13 @@ This guide is primarily written to accommodate installation on upstream Kubernet
       <td>yes</td>
       <td>yes</td>
     </tr>
+    <tr>
+      <td>v1.0.0</td>
+      <td>no</td>
+      <td>no</td>
+      <td>yes</td>
+      <td>yes</td>
+    </tr>
```
Comment (rgcostea) [inline README.md]: Drop the beta version altogether.  Also change the table to be <= 1.12.

File: README.md [hunk 2]
```diff
@@ -126,6 +133,20 @@ kubectl create -f https://raw.githubusercontent.com/hpe-storage/co-deployments/m
...
+kubectl create -f https://raw.githubusercontent.com/hpe-storage/co-deployments/master/yaml/csi-driver/nimble-csp.yaml
+kubectl create -f https://raw.githubusercontent.com/hpe-storage/co-deployments/master/yaml/csi-driver/hpe-csi-k8s-1.16.yaml
+```
+
+Kubernetes 1.17
+```
+kubectl create -f https://raw.githubusercontent.com/hpe-storage/co-deployments/master/yaml/csi-driver/hpe-linux-config.yaml
+kubectl create -f https://raw.githubusercontent.com/hpe-storage/co-deployments/master/yaml/csi-driver/nimble-csp.yaml
+kubectl create -f https://raw.githubusercontent.com/hpe-storage/co-deployments/master/yaml/csi-driver/hpe-csi-k8s-1.17.yaml
+```
```
Comment (rgcostea) [inline README.md]: Drop 1.12 above.  Also, we can probably condense this down to a few lines.  The only difference is the k8s version.

File: README.md [hunk 3]
```diff
@@ -126,6 +133,20 @@ kubectl create -f https://raw.githubusercontent.com/hpe-storage/co-deployments/m
...
+
 Depending on which version being deployed, different API objects gets created.
 
 ## Using the HPE CSI Driver for Kubernetes
```
Comment (rgcostea) [inline README.md]: Please add a support.MD file stating what we will support for the 1.0 release.

**Comment by raunakkumar** [comment]
Will make the changes suggested. Thanks

**Pull request merged**